client/src/modules/templates/modals/report.save.html

```diff
@@ -0,0 +1,37 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static" translate>{{SaveCtrl.report.title_key}}</li>
+    <li class="title" translate>REPORT.UTIL.SAVE_AS</li>
+  </ol>
+</div>
+
+<form name="SaveForm" bh-submit="SaveCtrl.submit(SaveForm)" bh-form-defaults autocomplete="off" novalidate>
```

Comment:
I'm also not sure the autocomplete="off" will work - I think the `<input>`'s autocomplete="off" is sufficient.